Application/EdFi.Ods.Repositories.NHibernate.Tests/Modules/DomainModelModule.cs

```diff
@@ -22,30 +22,29 @@ public class DomainModelModule : Module
     {
         protected override void Load(ContainerBuilder builder)
         {
-            builder.RegisterType<DomainModelProvider>().As<IDomainModelProvider>().SingleInstance();
-
-            // Domain Model
-            builder.Register(c => c.Resolve<IDomainModelProvider>().GetDomainModel()).As<DomainModel>().SingleInstance();
+            builder.RegisterType<DomainModelProvider>().As<IDomainModelProvider>();
```

Comment:
the more i think about this, we should have a singleton for the domain model provider